README.md

```diff
@@ -1,19 +1,98 @@
-### Nightly Build Status (develop branch)
+# Hydroshare _(hydroshare)_
 
```

Comment:
OK.  For the header consider HydroShare, with S capitalized.  We've been doing this in writing, though not sure how important this is.